src/main/java/ch/njol/skript/util/SkriptColor.java

```diff
@@ -182,6 +174,11 @@ public static SkriptColor fromBukkitColor(org.bukkit.Color color) {
 		assert false;
 		return null;
 	}
+
+	public static Color fromBukkitOrRgbColor(org.bukkit.Color color) {
+		Color fromBukkit = fromBukkitColor(color);
+		return fromBukkit == null ? new ColorRGB(color.getRed(), color.getGreen(), color.getBlue()) : fromBukkit;
+	}
```

Comment:
I don't think this should be added here to the SkriptColor class. The SkriptColor class is for chat format, and general simple related colours.

There is a pull request to have `ColorRGB.fromBukkit(Color)`. I cannot find that pull right now. Essentially you see how ColorRGB is non static initialization, well the plan was to make it static like all the other colour classes for standardization.

This method makes no sense belonging in the SkriptColor class when the for mentioned changes are scheduled. It should always return ColorRGB, and belong in the ColorRGB class.